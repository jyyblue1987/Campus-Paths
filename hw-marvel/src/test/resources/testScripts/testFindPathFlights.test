# Test find path on flights.csv

LoadGraph flights flights.csv

FindPath flights GARUDA-INDONESIA EVA-AIRWAYS