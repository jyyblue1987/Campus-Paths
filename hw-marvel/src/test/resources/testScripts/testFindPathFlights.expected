# Test find path on flights.csv

loaded graph flights

path from GARUDA-INDONESIA to EVA-AIRWAYS:
GARUDA-INDONESIA to JAPAN-AIRLINES via 54
JAPAN-AIRLINES to QATAR-AIRWAYS via 10
QATAR-AIRWAYS to EVA-AIRWAYS via 8